pkg/executables/clusterctl_test.go

```diff
@@ -241,12 +243,47 @@ func TestClusterctlBackupManagement(t *testing.T) {
 			},
 			wantMoveArgs: []interface{}{"move", "--to-directory", fmt.Sprintf("%s/%s", clusterName, managementClusterState), "--kubeconfig", "", "--namespace", constants.EksaSystemNamespace},
 		},
+		{
+			testName: "existing backup",
+			cluster: &types.Cluster{
+				Name:           clusterName,
+				KubeconfigFile: "cluster.kubeconfig",
+			},
+			setup: func(t *testing.T) {
+				existingPath := filepath.Join(".", clusterName, managementClusterState)
+				err := os.MkdirAll(existingPath, os.ModePerm)
+				if err != nil {
+					t.Fatalf("Could not create directory: %s", err)
+				}
+				t.Cleanup(func() {
+					os.RemoveAll(existingPath)
+				})
+			},
```

Comment:
I think you can just use `t.TempDir()` as your root directory. It should be automatically be removed once the test is complete

Reply:
I ended up moving this to a helper function since t.TempDir() doesn't allow me to specify the actual path it's created at.